### PR TITLE
Add patches for spanish fan translations

### PR DESCRIPTION
**Pull request by MrWrightyLC**
This PR adds support for these Fan Translations
- Tales of the Abyss - Spanish Fan Translation by [Tales Translations](https://toa.tales-tra.com/).
WS
- Persona 3 FES - Spanish Fan Translation by [TraduSquare](https://tradusquare.es/proyectos/persona-3-fes/).
WS, 480p and Controllable Party Members

### CODE REVIEW

File: patches/SLUS-21386_A616A6C2.pnach [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+patch=1,EE,002B46C4,word,240601AA //Room Names X Position
+patch=1,EE,002A265C,word,3C034455 //Character Textbox Viewport Width (Out of Battle)
+patch=1,EE,002A5080,word,3C034455 //Character Textbox Viewport Width (In Battle)
+patch=1,EE,0030837C,word,3C013F95 //Portraits Zoom
+patch=1,EE,002CE478,word,3C034415 //Portraits Vert
+patch=1,EE,002CE4E0,word,3C024400 //Portraits Y Position
+patch=1,EE,005D6878,word,43D55555 //Skits Text Position
+patch=1,EE,002A3CF8,word,3C0543D5 //Get Item Textbox Position
+patch=1,EE,002A222C,word,3C074455 //Story Textbox Range Extend
+patch=1,EE,0011A668,word,00000000 //Bloom Off
```
Comment (pgert) [inline patches/SLUS-21386_A616A6C2.pnach:50]: Reported to be no longer needed:
 https://forums.pcsx2.net/Thread-PCSX2-Widescreen-Game-Patches?pid=627849#pid627849

File: patches/SLUS-21386_A616A6C2.pnach [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+patch=1,EE,2066CA00,extended,3C033F19 //Wanted Direction Shadow 2 Offset (World Map)
+patch=1,EE,2066CDB0,extended,3C033F19 //Wanted Direction Shadow 3 Offset (World Map)
+patch=1,EE,206A47A4,extended,240601AA //Level Names (Places) X Position (World Map)
+patch=1,EE,206A4A48,extended,240601AA //Level Names (Objects) X Position (World Map)
+patch=1,EE,206A3F08,extended,240601AA //Level Names (Alternate) X Position (World Map)
+patch=1,EE,206A47F8,extended,240602A1 //Enter (Places) X Position (World Map)
+patch=1,EE,206A4AC8,extended,240602A1 //Enter (Objects) X Position (World Map)
+patch=1,EE,206A3F9C,extended,240602A1 //Enter (Alternate) X Position (World Map)
+patch=1,EE,206AA610,extended,3C154480 //Fades X Position (World Map)
+patch=1,EE,206AA654,extended,3C134200 //Fades Size (World Map)
```
Comment (pgert) [inline patches/SLUS-21386_A616A6C2.pnach:108]: Add this:
patch=1,EE,205E570C,extended,00000000 // black grid fix - by sergx12
// https://forums.pcsx2.net/Thread-PCSX2-Widescreen-Game-Patches?pid=627849#pid627849

File: patches/SLUS-21386_B6B5A6DC.pnach [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+patch=1,EE,002B46C4,word,240601AA //Room Names X Position
+patch=1,EE,002A265C,word,3C034455 //Character Textbox Viewport Width (Out of Battle)
+patch=1,EE,002A5080,word,3C034455 //Character Textbox Viewport Width (In Battle)
+patch=1,EE,0030837C,word,3C013F95 //Portraits Zoom
+patch=1,EE,002CE478,word,3C034415 //Portraits Vert
+patch=1,EE,002CE4E0,word,3C024400 //Portraits Y Position
+patch=1,EE,005D6878,word,43D55555 //Skits Text Position
+patch=1,EE,002A3CF8,word,3C0543D5 //Get Item Textbox Position
+patch=1,EE,002A222C,word,3C074455 //Story Textbox Range Extend
+patch=1,EE,0011A668,word,00000000 //Bloom Off
```
Comment (pgert) [inline patches/SLUS-21386_B6B5A6DC.pnach:50]: same thing here

File: patches/SLUS-21386_B6B5A6DC.pnach [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+patch=1,EE,2066CA00,extended,3C033F19 //Wanted Direction Shadow 2 Offset (World Map)
+patch=1,EE,2066CDB0,extended,3C033F19 //Wanted Direction Shadow 3 Offset (World Map)
+patch=1,EE,206A47A4,extended,240601AA //Level Names (Places) X Position (World Map)
+patch=1,EE,206A4A48,extended,240601AA //Level Names (Objects) X Position (World Map)
+patch=1,EE,206A3F08,extended,240601AA //Level Names (Alternate) X Position (World Map)
+patch=1,EE,206A47F8,extended,240602A1 //Enter (Places) X Position (World Map)
+patch=1,EE,206A4AC8,extended,240602A1 //Enter (Objects) X Position (World Map)
+patch=1,EE,206A3F9C,extended,240602A1 //Enter (Alternate) X Position (World Map)
+patch=1,EE,206AA610,extended,3C154480 //Fades X Position (World Map)
+patch=1,EE,206AA654,extended,3C134200 //Fades Size (World Map)
```
Comment (pgert) [inline patches/SLUS-21386_B6B5A6DC.pnach:108]: same thing here

**Comment by refractionpcsx2** [comment]
Are these rom hacks?

**Pull request merged**